packages/core/integration-tests/test/html.js

```diff
@@ -1561,7 +1561,7 @@ describe('html', function() {
 
     // b.html hitting the parallel request limit should not prevent g.html from being optimized
     html = await outputFS.readFile(path.join(distDir, 'g.html'), 'utf8');
-    assert.equal(html.match(/<script/g).length, 2);
+    assert.equal(html.match(/<script/g).length, 1);
```

Comment:
Now that we are no longer creating single-bundle shared bundles, `g`'s bundle no longer includes the additional shared bundle it would have had for `lodash/difference` -- `b.html` hits the parallel request limit by the time we attempt to extract `lodash/difference` into a shared bundle

Reply:
Hmm I think this test intends to capture that though. Maybe add another bundle that has lodash/difference?